src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ListingDetailsSteps.java

```diff
@@ -75,7 +75,7 @@ public void iLookAtSEDDetails() {
     public void iLookAtAdditionalInformation() {
         WebElement link = ListingDetailsPage.additionalInfoAccordion(getDriver());
         ((JavascriptExecutor) getDriver()).executeScript("arguments[0].click();", link);
-        getWait().until(ExpectedConditions.visibilityOf(ListingDetailsPage.targetedUser(getDriver())));
+        getWait().until(ExpectedConditions.visibilityOf(ListingDetailsPage.meaningfulUseUsers(getDriver())));
```

Comment:
This is the only item that is guaranteed to be on every listing's detail's additional information. Definitely recommend keeping this step as is :)